recipes-graphics/wayland/weston-init.bbappend

```diff
@@ -15,29 +15,17 @@ SRC_URI_append_mx6sl = " file://weston.config"
 INI_UNCOMMENT_ASSIGNMENTS_append_imx = " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'x11 wayland', 'xwayland=true', '', d)} \
 "
-INI_UNCOMMENT_ASSIGNMENTS_append_mx6dl = " \
-    use-g2d=1 \
-"
-INI_UNCOMMENT_ASSIGNMENTS_append_mx6q = " \
-    use-g2d=1 \
-"
-INI_UNCOMMENT_ASSIGNMENTS_append_mx7ulp = " \
-    use-g2d=1 \
-"
 INI_UNCOMMENT_ASSIGNMENTS_append_mx8 = " \
     repaint-window=16 \
 "
-INI_UNCOMMENT_ASSIGNMENTS_append_mx8mm = " \
-    use-g2d=1 \
-"
-INI_UNCOMMENT_ASSIGNMENTS_append_mx8mp = " \
-    use-g2d=1 \
-"
 INI_UNCOMMENT_ASSIGNMENTS_append_mx8mq = " \
     gbm-format=argb8888 \
     \\[shell\\] \
     size=1920x1080 \
 "
+INI_UNCOMMENT_ASSIGNMENTS_append_imxgpu2d = " \
+    use-g2d=1 \
+"
```

Comment:
```suggestion
INI_UNCOMMENT_ASSIGNMENTS_append_imxgpu2d = " \
    use-g2d=1 \
"

# FIXME: The 8QM and 8QXP SoCs have better performance without G2D so don't enable it
# Ideally, this should be seamless and Vivante ought to handle it internally and take the fastest
# rendering code.
INI_UNCOMMENT_ASSIGNMENTS_remove_mx8qm = "use-g2d=1"
INI_UNCOMMENT_ASSIGNMENTS_remove_mx8qxp = "use-g2d=1"
```

Reply:
@thochstein does this help?

Reply:
Yes, but I learned recently that doing a `_remove` is very difficult to undo in dependent layers, so perhaps there should be another variable?

```
INI_UNCOMMENT_USE_G2D ?= "use-g2d=1"
INI_UNCOMMENT_USE_G2D_mx8qm = ""
INI_UNCOMMENT_USE_G2D_mx8qxp = ""
```

Reply:
So this?

INI_UNCOMMENT_USE_G2D ?= "use-g2d=1"
INI_UNCOMMENT_USE_G2D_mx8qm = ""
INI_UNCOMMENT_USE_G2D_mx8qxp = ""
INI_UNCOMMENT_ASSIGNMENTS_append_imxgpu2d = " \    
${INI_UNCOMMENT_USE_G2D} \
"

Reply:
Yes, that was my thinking.

Reply:
@chrisdimich please include a detailed `FIXME` comment explaining it. However, it can be added as:

```
INI_UNCOMMENT_USE_G2D_imxgpu2d ?= "use-g2d=1"
INI_UNCOMMENT_USE_G2D_mx8qm = ""
INI_UNCOMMENT_USE_G2D_mx8qxp = ""
INI_UNCOMMENT_ASSIGNMENTS_append_imxgpu2d = " \
${INI_UNCOMMENT_USE_G2D}
"
```